### PR TITLE
Update en.yaml based on TOC feedback.

### PR DESCRIPTION
**Pull request by aijamalnk**
Updating Falco's landing page with feedback from CNCF's TOC. Rephrasing a few things.

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
TOC gave specific feedback about some of the wording on the site. I am editing to match here.

**Which issue(s) this PR fixes**:
Related to https://github.com/falcosecurity/falco-website/issues/1003

### CODE REVIEW

**Comment by aijamalnk** [comment]
@nate-double-u @incertum fyi

File: i18n/en/en.yaml [hunk 1]
```diff
@@ -67,9 +67,9 @@ home_show_more:
 home_show_less:
   other: Show less
 home_falco_what_is_falco_title:
-  other: Detect and respond to threats in real time
+  other: Detect threats in real time
 home_falco_what_is_falco:
-  other: Falco is the open source standard for runtime security for hosts, containers, Kubernetes and the cloud. Get real-time visibility into unexpected behaviors, config changes, intrusions, and data theft.
+  other: Falco is the open source solution for runtime security for hosts, containers, Kubernetes and the cloud. Get real-time visibility into unexpected behaviors, config changes, intrusions, and data theft.
```
Comment (incertum) [inline i18n/en/en.yaml:72]: `Falco is a widely adopted open source solution ...`?

Stay with `abnormal behaviors` consistently?

proposed rewrite: 

`Get real-time visibility into abnormal behaviors and potential security threats, intrusions, and data theft or compliance violations.`

File: i18n/en/en.yaml [hunk 2]
```diff
@@ -67,9 +67,9 @@ home_show_more:
 home_show_less:
   other: Show less
 home_falco_what_is_falco_title:
-  other: Detect and respond to threats in real time
+  other: Detect threats in real time
```
Comment (incertum) [inline i18n/en/en.yaml:70]: `Detect security threats in real time`?

File: data/en/usecases.yaml [hunk 3]
```diff
@@ -1,8 +1,8 @@
 - title: Container Security
-  description: Secure containerized applications, no matter what scale, using the power of eBPF.
+  description: Detect malicious behavior in containerized applications, no matter what scale, using the power of eBPF.
   url: /about/use-cases/#protect-containers-kubernetes-and-cloud-native-applications
 - title: Host Security
-  description: Protect your applications in real time wherever they run, whether bare metal or VMs.
+  description: Oversee your applications' behavior in real time wherever they run, whether bare metal or VMs.
```
Comment (incertum) [inline data/en/usecases.yaml:5]: Maybe just personal preference to consistently use the most commonly used terms, e.g. here `Monitor your ...`

File: data/en/features.yaml [hunk 4]
```diff
@@ -13,7 +13,7 @@
   - Streaming approach enables real-time response while minimizing storage costs and complexity.
   - Ready out-of-the-box with rules, which you can customize for your environment.
 - title: Open source
-  description: A multi-vendor and broadly supported standard that you can rely on.
+  description: A multi-vendor and broadly supported solution that you can rely on.
```
Comment (incertum) [inline data/en/features.yaml:16]: `A multi-vendor and widely adopted solution that you can rely on.`?

File: i18n/en/en.yaml (hunk 1 above)
Comment (aijamalnk) [reply i18n/en/en.yaml]: added thanks!

File: i18n/en/en.yaml (hunk 2 above)
Comment (aijamalnk) [reply i18n/en/en.yaml]: added thanks!

File: data/en/features.yaml (hunk 4 above)
Comment (aijamalnk) [reply data/en/features.yaml]: added thanks!

**Review by leogr — APPROVED** [review]
LGTM!

Thank you :pray:

**Pull request merged**